$OpenBSD: patch-configure_in,v 1.13 2015/12/23 09:19:57 dcoppa Exp $

setuid root/setgid utmp are handled by PLIST

Do not enable VTE support if libvte is around

--- configure.in.orig	Tue Dec 15 13:25:01 2015
+++ configure.in	Wed Dec 23 09:19:54 2015
@@ -675,44 +675,14 @@ AC_ARG_ENABLE(pty_helper,
 	pty_helper=$enable_pty_helper)
 if test "$utmp" = "yes" -a "$pty_helper" != "yes" ; then
 	UTMP_CFLAGS="-DUSE_UTMP"
-	has_utmp=`grep utmp /etc/group 2>/dev/null`
-	if test "$has_utmp" ; then
-		INSTALL_OPT="-m 2755 -g utmp"
-	else
-		INSTALL_OPT="-m 4755 -o root"
-	fi
-else
-	INSTALL_OPT="-m 755"
 fi
 
-if test "$gui" = "fb" ; then
-	INSTALL_OPT="-m 4755 -o root"
-fi
+INSTALL_OPT="-m 0555 -s"
 
 AC_SUBST(UTMP_CFLAGS)
 AC_SUBST(INSTALL_OPT)
 
-if test "${have_gtk}" = "yes" ; then
-	if test "${gtk_version}" = "3.0" ; then
-		PKG_CHECK_MODULES(VTE, vte-2.91, have_vte=yes, have_vte=no)
-		if test "$have_vte" = "yes" ; then
-			VTE_ABI=2.91
-		else
-			PKG_CHECK_MODULES(VTE, vte-2.90, have_vte=yes, have_vte=no)
-			if test "$have_vte" = "yes"; then
-				VTE_ABI=2.90
-			fi
-		fi
-	else
-		PKG_CHECK_MODULES(VTE, vte, have_vte=yes, have_vte=no)
-	fi
-	if test "$have_vte" = "yes" ; then
-		OUTPUT_FILES="gtk/Makefile ${OUTPUT_FILES}"
-		AC_SUBST(VTE_ABI)
-	fi
-else
-	have_vte=no
-fi
+have_vte=no
 
 tools="mlclient,mlconfig,mlcc,mlterm-menu,mlimgloader,registobmp"
 AC_ARG_WITH(tools,
